models/marts/web_events/snowplow_sessions_final.sql

```diff
@@ -0,0 +1,110 @@
+{{
```

Comment:
i think for consistency in naming conventions we may want to rename this to fct_snowplow_sessions